deprecation.rst

```diff
@@ -17,3 +17,18 @@ no version bumps are expected.
 
 The current list of deprecated EAPIs is stored as ``eapis-deprecated``
 in ``metadata/layout.conf``.
+
+
+.. index:: EAPI; banned
+
+Banned EAPIs
+------------
+:PG: 1002
+:Source: Council
+:Reference: https://gitweb.gentoo.org/repo/gentoo.git/tree/metadata/layout.conf
+:Reported: by pkgcheck and repoman
+
+Banned EAPIs must not be used by any ebuilds in the repository.
+
+The current list of banned EAPIs is stored as ``eapis-banned``
+in ``metadata/layout.conf``.
```

Comment:
This is not correct, the list in layout.conf is incomplete.

I think the best you can get is Council meeting summaries as authoritative source, and https://wiki.gentoo.org/wiki/Project:Package_Manager_Specification#Council_approval_and_use_in_Gentoo_repository for an overview.